docs/modules/ROOT/pages/cops_style.adoc

```diff
@@ -3416,6 +3416,11 @@ which are passed as arguments to those methods:
 The reason is that _unannotated_ format is very similar
 to encoded URLs or Date/Time formatting strings.
 
+When `MaxUnannotatedPlaceholdersAllowed` option is greater than zero,
```

Comment:
I think you have two starts of this sentence and you can probably delete this line, as it seems redundant.